mne/bem.py

```diff
@@ -863,6 +904,10 @@ def fit_sphere_to_headshape(info, dig_kinds=(FIFF.FIFFV_POINT_EXTRA,),
                 '%0.1f %0.1f %0.1f mm' % tuple(origin_head))
     logger.info('Origin device coordinates:'.ljust(30) +
                 '%0.1f %0.1f %0.1f mm' % tuple(origin_device))
+    if units == 'm':
```

Comment:
`m` and `mm` are pretty much the only distance units used in mne-python. `m` should really be preferred because it's an SI unit and used most often, but here we should probably keep a `mm` option since it's been there for a long time.